lib/valueptr.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Cppcheck - A tool for static C/C++ code analysis
+ * Copyright (C) 2007-2019 Cppcheck team.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef GUARD_VALUEPTR_HPP
+#define GUARD_VALUEPTR_HPP
```

Comment:
this include guard does not follow our style.

Reply:
The name `valueptrH` would follow our style. For information, this naming scheme was originally chosen because that will work best in the codegear C++ Builder IDE ... but well as far as I know, no Cppcheck developer has used that IDE for many years so it's not very important anymore.